medusajs/medusa
# docs: update auth docs + add new storefront guides

- Update auth docs with latest changes to authentication flows
- Add documentation for Google and GitHub providers
- Add storefront guides for social-login (using Google as an example)
- Update the OAS for auth routes.

> Note: This also includes documentation updates based on PR #9013. So, it should only be merged after that one is merged and published.
